packages/core/src/Services/socket-general.js

```diff
@@ -246,6 +245,7 @@ const BinarySocketGeneral = (() => {
 
     const authorizeAccount = response => {
         client_store.responseAuthorize(response);
+        WS.subscribeWebsiteStatus(ResponseHandlers.websiteStatus);
```

Comment:
I needed to call `website_status` after authorize response here as if it is called before the response of `authorize` user will not get `p2p_config` flag